Dockerfile

```diff
@@ -10,6 +10,8 @@ RUN CGO_ENABLED=0 go build -o /go/bin/app -ldflags="-X 'main.version=`git rev-pa
 FROM alpine:3.19
 
 COPY --from=build /go/bin/app /
+COPY --from=0 /usr/local/go/lib/time/zoneinfo.zip /
```

Comment:
IIUC, `--from=0` mean the same as `--from=build` in this context. I think `--from=build` is more explicit: it doesn't require to know about internal Docker convention.

Reply:
👍  agreed